examples/jsm/loaders/GLTFLoader.js

```diff
@@ -638,6 +645,38 @@ class GLTFMaterialsUnlitExtension {
 
 }
 
+/**
+ * Materials Emissive Strength Extension
+ *
+ * Specification: https://github.com/KhronosGroup/glTF/blob/5768b3ce0ef32bc39cdf1bef10b948586635ead3/extensions/2.0/Khronos/KHR_materials_emissive_strength/README.md
+ */
+ class GLTFMaterialsEmissiveStrengthExtension {
+
+	constructor( parser ) {
+
+		this.parser = parser;
+		this.name = EXTENSIONS.KHR_MATERIALS_EMISSIVE_STRENGTH;
+
+	}
+
+	extendMaterialParams( materialIndex, materialParams ) {
+
+		const parser = this.parser;
+		const materialDef = parser.json.materials[ materialIndex ];
+
+		if ( ! materialDef.extensions || ! materialDef.extensions[ this.name ] ) {
+
+			return Promise.resolve();
+
+		}
+
+		materialParams.emissiveIntensity = materialDef.extensions[this.name].emissiveStrength;
```

Comment:
Nit: the `emissiveStrength` property is optional. It probably doesn't matter but let's avoid setting the material parameter to `undefined` here.

```suggestion
		const emissiveStrength = materialDef.extensions[this.name].emissiveStrength;
		
		if ( emissiveStrength !== undefined ) {

			materialParams.emissiveIntensity = emissiveStrength;

		}

```

Reply:
Good point, thank you!